optimum/exporters/onnx/__main__.py

```diff
@@ -17,6 +17,7 @@
 import argparse
 from pathlib import Path
 
+from requests.exceptions import ConnectionError as RequestsConnectionError
```

Comment:
Why changing the name?

Reply:
It collides with python native ConnectionError